### ISSUE TITLE
Missing license

### ISSUE THREAD

**Issue by kesslermaximilian** [Gargantuar314/university#3]
You did not (yet) provide a license for these lecture notes.

I would very much appreciate an appropriate license to answer questions like these:

- Is this free software?

and if not:

- How can people contribute to this?
- Are others allowed to redistribute your notes?
- Are others allowed to edit your notes or reuse them for their purposes, i.e. alter your document?
- ...

If you are unsure about this and do not intend to sell these in the future, I would recommend looking e.g. at
- GPLv3 (Gnu General Public License)
- The LPPL (LaTeX project public license)
- The MIT license

Note the first one is copyleft and the others not.
For more info you could e.g. visit https://wiki.civiccommons.org/Choosing_a_License/.

**Comment by Gargantuar314** [comment]
That's true. By default, if I don't provide a license, I hold exclusive copyright for the whole project.

The reason why I haven't provided a license yet is, that I might get a side job from my university next year to digitalise the lectures. Maybe I'll use parts of `ana1` and sell it to my university, but let's see.

If this shouldn't be the case, I'd provide the code under the Unlicense and the content of the PDFs under public domain/Creative Commons Zero License. (In fact, I could provide the project under a very free license and still sell it to others 🙃.
